lib/govuk_publishing_components/presenters/contextual_navigation.rb

```diff
@@ -37,7 +37,7 @@ def breadcrumbs
             }
           ]
         else
-          nav_helper.breadcrumbs[:breadcrumbs]
+          GovukNavigationHelpers::Breadcrumbs.new(content_item).breadcrumbs[:breadcrumbs]
```

Comment:
Any reason why we don't memoize breadcrumbs?  We do everything else in this class.  Doesn't stop a merge, just a question